ui/pages/_app.js

```diff
@@ -439,7 +446,7 @@ class MesheryApp extends App {
                     maxSnack={10}
                   >
                     <MesheryProgressBar />
-                    <Header onDrawerToggle={this.handleDrawerToggle} />
+                    <Header onDrawerToggle={this.handleDrawerToggle} isDrawerCollapsed={isDrawerCollapsed}/>

```

Comment:
Use separate names for prop and the state.
You can use `onDrawerCollapse` for prop, and keep the state name as it is.